.eslintrc.js

```diff
@@ -20,13 +20,14 @@ module.exports = {
   rules: {
     "jsdoc/check-types": 2,
     "jsdoc/newline-after-description": 2,
+    'indent': ['error', 2],
```

Comment:
What's this being added for?

Reply:
Oh, now that I'm looking at the rest of the changes, I see what you're doing. But technically this isn't consistent with Google style because in some cases Google style recommends indenting 4 spaces (e.g. line continuations and stuff like that).

The google eslint package v0.10.0 adds an indentation rule more inline with what's recommended by Google JS style. It's probably worth just upgrading to that.

Reply:
As discussed in person, there are going to be a lot of additional files that need whitespace adjustment if we don't include this.

We're already using `"eslint-config-google": "^0.10.0"` as part of this PR.

Let's do separate PR to remove this override and get all the whitespace consistent rather than including any more changes in this PR.

Reply:
SGTM